module/VuFind/src/VuFind/Search/EDS/AbstractEDSParams.php

```diff
@@ -91,4 +99,24 @@ public function getView()
         $viewArr = explode('|', $this->view ?? '');
         return $viewArr[0];
     }
+
+    /**
+     * Don't actually parse the operator.  Use $this->getFacetOperator();
+     *
+     * @param string $field Prefixed string
+     *
+     * @return array (0 = operator, 1 = field name)
+     */
+    protected function parseOperatorAndFieldName($field)
+    {
+        $field = str_replace('~', '', $field);
+
+        if (array_search($field, $this->FORCED_OR_FIELDS)) {
```

Comment:
This should be `in_array` -- otherwise, a match with the first array element will return an index of 0, which will evaluate to false.